jinahub/encoders/text/DPRTextEncoder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:2.0.12-perf
+FROM jinaai/jina:2.0-py38-perf
```

Comment:
what is this docker image, why sometimes it's py37 sometimes is py38?

Reply:
Ok good point, when doing refactoring we will decide on a common docker image here

Reply:
Now it's all over the place

Reply:
well this is a problem with any image you might choose to use. Needs to be fixed with dependabot, I'd say. out of scope on deciding / testing in this PR